python/rikai/types/geometry.py

```diff
@@ -170,6 +181,59 @@ def __eq__(self, o: Box2d) -> bool:
             self.to_numpy(), o.to_numpy()
         )
 
+    @staticmethod
+    def _verified_scale(
+        scale: Union[int, float, Tuple]
+    ) -> Tuple[Number, Number]:
+        if isinstance(scale, Number):
+            assert scale > 0, f"scale must be positive, got {scale}"
+            return scale, scale
+        assert (
+            type(scale) == tuple and len(scale) == 2
+        ), "scale must be either a number or a 2-element tuple"
+        assert (
+            scale[0] > 0 and scale[1] > 0
+        ), f"scale must be positive, got {scale}"
+        return scale
+
+    def __truediv__(self, scale: Union[int, float, Tuple]) -> Box2d:
```

Comment:
nit: when both `int` and `float` are acceptable, you can use just `float` if I'm reading this correctly: https://www.python.org/dev/peps/pep-0484/#the-numeric-tower
```suggestion
    def __truediv__(self, scale: Union[float, Tuple[float, float]]) -> Box2d:
```